src/app/+display/section/section.component.html

```diff
@@ -1,7 +1,41 @@
-<div class="card-header font-weight-bold text-primary text-capitalize">{{section.name}}</div>
+<div class="card-header font-weight-bold text-primary text-capitalize">
+  {{section.name}}
+  <div class="float-right" *ngIf="isShareable()">
+    <div id="embedDropdown" class="dropdown d-inline-block" scholarsDropdown>
+      <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+      </button>
+      <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+        <form>
+          <div class="form-group col-12">
+            <label>Embeddable Widget</label>
+            <div class="input-group">
+              <input class="form-control" type="text" ng-bind-html value="{{buildEmbedCode()}}" #embedCode/>
+              <span class="input-group-btn">
+                <button type="button" class="btn btn-default" (click)="copyToClipBoard(embedCode)">Copy</button>
+              </span>
+            </div>
+          </div>
+        </form>
+      </div>
+    </div>
+  </div>
+</div>
 <div class="card-body">
   <div [innerHtml]="section.templateFunction(document) | safeHtml"></div>
   <div *ngFor="let subsection of section.subsections">
     <scholars-subsection [subsection]="subsection" [document]="document"></scholars-subsection>
   </div>
-</div>
\ No newline at end of file
+</div>
+
+<style>
```

Comment:
Styles should not be added to the html template. They should be added in the components SCSS file. In this case `section.component.scss`.